content/jobs/django2020.md

```diff
@@ -0,0 +1,31 @@
+---
+position: open
+banner: Django Developer
+sub: Leading our back-end development process
+date: 2020-09-23
+---
+
+OddBird is looking for a **Django Developer** to join the team,
+and we'd love to hear from you!
+
+## You:
+
+- Have experience working with Django, and Python more generally
```

Comment:
@wlonk @carljm What do you two think about whether we want to advertise for a Django dev vs a Python dev? I notice that in 2016 (for Kit's position) we didn't mention Django at all -- I'm assuming we do want Django experience though?

https://github.com/oddbird/oddsite/blob/e4465df3a3778b73b27ff7a88da8fdec9d24fcae/content/oddjob.rst

Reply:
Pro-Django:
 - All our existing projects are Django-based, and so it's pretty truthful about the work someone'd be doing at the outset.

Con-Django:
 - It limits the pool more; folks who might think "I know Python well, and I could learn Django, but I don't know it yet" might self-select out.

Reply:
I might be inclined to mention Django but put more emphasis on Python (Eg "experience working with Python for the web, and ideally Django") But I don't feel strongly about it.

Reply:
Yes, that! That's the perfect phrasing, imo.